demos/try17_AffineModel.py

```diff
@@ -0,0 +1,114 @@
+# %%
+import numpy as np
+import matplotlib.pyplot as plt
+import sys
+
+from cuqi.model import AffineModel, LinearModel
+from cuqi.experimental.mcmc import MH, LinearRTO
+from cuqi.distribution import Gaussian, LMRF, GMRF, JointDistribution
+from cuqi.geometry import Discrete, Continuous1D
+from cuqi.array import CUQIarray
+
+#%% Test problem 
+### y = Ax + b
+### Infer x, given the model A and b, and noisy observations of y
```

Comment:
I actually disagree here. The "b" is the bias/shift. A part of the forward model. There are two ways to incorporate that, either you use some algebra and realize that you can subtract the bias from the data, or you define an affine model and let CUQIpy handle it.

Reply:
Got it! Thanks for the clarification. It could be useful to mention this at the top to clarify this is the motivation behind showcasing the two approaches.

e.g. something along what you mentioned:
```
# %% Test problem
### y = Ax + b
### Infer x, given the model A and shift/bias b, and noisy observations of y
### There are two ways to solve this problem: either you use the linear model only y=Ax realizing that you can subtract the bias from the data, or define an affine model and let CUQIpy handle it.```

Reply:
Done